src/Tribe/Aggregator/Cron.php

```diff
@@ -192,18 +192,21 @@ public function action_register_cron() {
 	 * Instead of having cron be scheduled to specific times, we will check every 30 minutes
 	 * to make sure we can insert without having to expire cache.
 	 *
-	 * @param  array $schedules
+	 * @param array $schedules Current list of schedules.
 	 *
-	 * @return array
+	 * @return array            Modified list of schedules.
 	 */
-	public function filter_add_cron_schedules( array $schedules ) {
+	public function filter_add_cron_schedules( $schedules = [] ) : array {
```

Comment:
```suggestion
	public function filter_add_cron_schedules( $schedules = [] ) {
```
That's a php 7+ syntax - unless we're now requiring php 7+ we shouldn't do this?

Reply:
Correct, thanks @Camwyn